internal/server/statvar/fetcher/formula_fetcher.go

```diff
@@ -39,83 +41,108 @@ func FetchFormulas(
 	remoteRespChan := make(chan *pbv2.NodeResponse, 1)
 	// Fetch for BT and SQL.
 	errGroup.Go(func() error {
-		data, _, err := v1pv.Fetch(
-			errCtx,
-			store,
-			[]string{"StatisticalCalculation"},
-			[]string{"typeOf"},
-			0,
-			"",
-			"in",
-		)
-		if err != nil {
-			return err
-		}
 		statCal := []string{}
```

Comment:
nit: these are calculation dcids, correct? Might be worthwhile to name it so and / or add a comment.

Reply:
Good idea, I added "dcid" to the name